docusaurus/docs/quickstarts/zac/installation.md

```diff
@@ -13,7 +13,9 @@ and can be accomplished in two basic ways shown below.
 
 It's expected that you're using `bash` for these commands. If you're using Windows we strongly recommend that you install 
 and use Windows Subsystem for Linux (WSL). Other operating systems it's recommended you use `bash` unless you are able to 
-translate to your shell accordingly. 
+translate to your shell accordingly.
+
+You will need `node` and `npm` executables. You may need a newer version of Node than your OS package manager provides. Ubuntu Jammy currently provides Node v12 which is too old. If the app complains about `await` then a newer version is needed. In that case, you could use [Node Version Manager](https://github.com/nvm-sh/nvm#readme) (`nvm`) instead of `apt` to install `node` and `npm`.

```

Comment:
i understand the desire to be helpful, but i would just express that node 16+ is needed. go figure out how to install that

Reply:
> You will need `node` and `npm` executables from Node.js v16+. Ubuntu Jammy currently provides v12.